### DISCUSSION TITLE
Welcome to swiftui Discussions!

### DISCUSSION THREAD

**Discussion by ygit** [ygit/swiftui#25] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by philipturner** [comment]
Would you mind starring my repository, [ARHeadsetKit](https://github.com/philipturner/ARHeadsetKit)? I’m trying to reach 15 stars so I can get on the [awesome-swift](https://github.com/matteocrippa/awesome-swift) list.

**Reply by ygit** [reply]
Done @philipturner
